app/gui2/src/components/GraphEditor/widgets/WidgetFunction.vue

```diff
@@ -157,8 +157,9 @@ const inheritedConfig = computed(() => {
   if (props.input.dynamicConfig?.kind === 'OneOfFunctionCalls' && methodCallInfo.value != null) {
     const cfg = props.input.dynamicConfig
     const info = methodCallInfo.value
-    const name = entryQn(info?.suggestion)
-    return cfg.possibleFunctions.get(name)
+    const fullName = entryQn(info?.suggestion)
+    const autoscopedName = '..' + info?.suggestion.name
+    return cfg.possibleFunctions.get(fullName) ?? cfg.possibleFunctions.get(autoscopedName)
```

Comment:
Should we always check autoscoped name here, or is it limited only to cases where the suggetsion is an enum constructor?

Also I think this logic change should be included in `possibleFunctions` property documentation. It no longer typically contains only FQNs.

Reply:
> Should we always check autoscoped name here, or is supposed to be limited only to cases where the suggestion is an enum constructor?

There is no harm in doing it every time we can.

> Also I think this logic change should be included in possibleFunctions property documentation. It no longer typically contains only FQNs.

Added note there.